storybook/components/StoryInlineNotification/StoryInlineNotification.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from "react"
+import {
+  InlineNotification,
+  InlineNotificationProps,
+} from "~components/Notification"
+
+export type StoryInlineNotificationProps = {
+  content: ReactNode
+} & InlineNotificationProps
+
+export const StoryInlineNotification = ({
```

Comment:
Same comments as the other PR, but I reckon just make this one a contextual component. Something like `SbNotificationUpcomingRelease`.

Reply:
Why make it specific?

Reply:
Re-linking posted in other [PR](https://github.com/cultureamp/kaizen-design-system/pull/4395#discussion_r1414893197)

Reply:
Also what do you think of the name `SbReleaseNotification`. I think it still captures the immediacy / intent when it should be used

Reply:
Yeah open to any naming :)

Reply:
Groovy that's all updated a ready to roll :)